### PR TITLE
Added First Name to Member Search

### PR DESCRIPTION
**Pull request by sarangtc**
Added First Name to Member Search

### CODE REVIEW

**Comment by sarangtc** [comment]
image

**Comment by 375michael40veit** [comment]
@sarangtc @Larian77 
It would not be a problem to implement the search by first name, but I have not yet fully understood the scenario. In which situation is it helpful to search by first name only?
Searching by first name and surname would be more effective in my opinion. However, since I don't know your situation in detail, could you explain it in more detail?

**Comment by sarangtc** [comment]
I agree searching by both (or whichever is filled) would be better.
In India, surnames are very very common and a starting with surname search is sometimes useless.
We need to decide:

1. Whether to add and/or first name search
2. Whether to keep starting with, or give the user the ability to choose starting with/any part of field.

**Comment by Larian77** [comment]
I think if there are areas where the first name search is actually more useful, we should implement it, but the question is whether we will do it in a future version (I would also like to have 0.8.2 again for smaller things, I already have a few ideas, more after my holiday, but something like that would also fit in there). The question is whether we should not use a field here where you can select what you want to use as a search via a dropdown like when searching for a medium.

Basically, I would also like to be able to search everywhere for parts of the word, which is the usual way for most users on the net.

**Comment by prschopf** [comment]
Searching for the member first name is a great feature for me. I already instaled this mod and Its working fine. 

I also changed all reports and víews to show members first name+last name because we use in this way.

**Comment by LeAchim59** [comment]
@Larian77 
Many thanks to @sarangtc , I also find the option very useful.
I support the implemented version with a separate input field. Separate input fields should be provided for the most common search options so that you don't have to make a selection every time. The search is also faster ;-)
